### PR TITLE
set Cloud SDK location when creating a custom runtime in the wizard

### LINKED ISSUE #271: created runtime does not include a cloud SDK location
The runtime (custom) that is created when a new project is created does not include a cloud SDK location. The server will not run unless this is manually configured.

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis fix #271

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java [hunk 1]
```diff
@@ -136,14 +139,18 @@ void installAppEngineRuntime(IFacetedProject project, IProgressMonitor monitor)
...
-      IRuntimeType appEngineRuntimeType =
-          ServerCore.findRuntimeType(DEFAULT_RUNTIME_ID);
+      IRuntimeType appEngineRuntimeType = ServerCore.findRuntimeType(DEFAULT_RUNTIME_ID);
       if (appEngineRuntimeType == null) {
         throw new NullPointerException("Could not find " + DEFAULT_RUNTIME_NAME + " runtime type");
       }
 
       IRuntimeWorkingCopy appEngineRuntimeWorkingCopy 
           = appEngineRuntimeType.createRuntime(null, monitor);
+      File sdkLocation = PathResolver.INSTANCE.getCloudSdkPath().toFile();
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: This should be `new CloudSdkProvider(null).getSdkLocation()`, right.

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: done

**Comment by elharo** [comment]
ping

**Comment by elharo** [comment]
PTAL @briandealwis

**Comment by briandealwis** [comment]
LGTM

**Pull request merged**